packages/js/src/Modules/Verto/BrowserSession.ts

```diff
@@ -361,10 +361,46 @@ export default abstract class BrowserSession extends BaseSession {
     return this._videoConstraints;
   }
 
+  /**
+   * Disables use of the webcam in subsequent calls.
+   *
+   * Note: The method will disable the video even though `video: true` is specified.
```

Comment:
Adding on to Hugo's suggestion:
```suggestion
   * Note: This method will disable the video even if `video: true` is specified.
```